python/setup.py

```diff
@@ -75,6 +77,59 @@ def run_install():
         print("###############################################################################################")
 
 
+def can_compile_with_openmp(cc, flags, gomp_paths):
+    tmpdir = tempfile.mkdtemp()
+    curdir = os.getcwd()
+    os.chdir(tmpdir)
+
+    # attempt to compile a test program with openmp
+    test_filename = 'test_openmp.c'
+    with open(test_filename, 'w') as f:
+        f.write('int main(void) { return 0; }');
+
+    compilation_cmd = [cc] + flags
+    for path in gomp_paths:
+        compilation_cmd  += ['-L' + path]
+    compilation_cmd += [test_filename]
+
+    try:
+        with open(os.devnull, 'w') as fnull:
+            exit_code = subprocess.call(compilation_cmd,
+                stdout=fnull, stderr=fnull)
+    except OSError:
+        exit_code = 1
+
+    # clean up
+    os.chdir(curdir)
+    shutil.rmtree(tmpdir)
+    return exit_code == 0
+
+
+class build_ext_scs(build_ext):
+    def build_extensions(self):
+        # fetch the name of the compiler to be invoked
+        cc = self.compiler.compiler_so[0]
```

Comment:
It appears that this line is breaking the build on windows:

https://ci.appveyor.com/project/bodono/scs/build/1.0.304/job/3uqr6dxjy759wkyt

"AttributeError: MSVCCompiler instance has no attribute 'compiler_so'"